coda-src/kerndep/coda.h

```diff
@@ -663,6 +664,7 @@ struct coda_open_by_fd_in {
 struct coda_open_by_fd_out {
     struct coda_out_hdr oh;
     int fd;
+    uint16_t venus_flags;
```

Comment:
How is this compatible with kernel modules that do not have access intents?
We need to think of a different solution here, maybe something when we initially open /dev/cfs0, similar to or piggybacked on the version ioctl.

Reply:
You are right. I partially tested it and worked but I don't think it's safe. I'll think about it.

Reply:
I just tested it as it is with only kernel module. It does work. But as I said I'm not sure it's safe. It works because the size of `union inputArgs` is bigger than the open-related structs. So basically the value is passes within allocated memory and ignored. What are your thoughts?

Reply:
And according to

```
	/* Op went through, interrupt or not... */
	if (req->uc_flags & CODA_REQ_WRITE) {
		out = (union outputArgs *)req->uc_data;
		/* here we map positive Venus errors to kernel errors */
		error = -out->oh.result;
		*outSize = req->uc_outSize;
		goto exit;
	}
```

in `upcall.c` we even obtain the size from the read buffer.